autopush/main.py

```diff
@@ -225,11 +234,13 @@ def make_settings(args, **kwargs):
         datadog_app_key=args.datadog_app_key,
         datadog_flush_interval=args.datadog_flush_interval,
         hostname=args.hostname,
+        db_host=args.db_host,
+        db_port=args.db_port,
+        db_secure=args.db_secure,
         statsd_host=args.statsd_host,
         statsd_port=args.statsd_port,
         pingConf=pingConf,
         router_tablename=args.router_tablename,
-        storage_tablename=args.storage_tablename,
```

Comment:
Parallelly, I am also considering if boto should be able to handle this
directly through the `~/.boto` config. This would make things seamless for
`autopush`.

On Wed, Jun 3, 2015 at 10:35 PM, Rohan Jain <crodjer@gmail.com> wrote:

> Yes, I did not intend to do this, will fix this.
>
> On Wed, Jun 3, 2015 at 10:20 PM, JR Conlin <notifications@github.com>
> wrote:
>
>> In autopush/main.py
>> <https://github.com/mozilla-services/autopush/pull/88#discussion_r31643753>
>> :
>>
>> >          statsd_host=args.statsd_host,
>> >          statsd_port=args.statsd_port,
>> >          pingConf=pingConf,
>> >          router_tablename=args.router_tablename,
>> > -        storage_tablename=args.storage_tablename,
>>
>> storage_tablename is still in use. Was this accidental?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/mozilla-services/autopush/pull/88/files#r31643753>.
>>
>
>
>
> --
> Thanks
> Rohan Jain
>



-- 
Thanks
Rohan Jain